feat: document `Masterdata.Person.Create` and `Masterdata.Person.Update` (#7)
Garaio-REM/remcloud-message-bus-api: masterdata_context.md

&nbsp;&nbsp;`maritalStatusCode` | `string` | a value of the marital status code table (see code table entries "Zivilstände" for valid codes). Send _either_ `maritalStatus` _or_ `maritalStatusCode` but not both.
&nbsp;&nbsp;`jobTitle` | `string` | job title
&nbsp;&nbsp;`dateOfBirth` | `string` | ISO 8601 encoded date, eg `'2019-03-01'`; **required**
&nbsp;&nbsp;`dateOfDeath` | `string` | ISO 8601 encoded date, eg `'2019-03-01'`
&nbsp;&nbsp;`homeTown` | `string` | home town
&nbsp;&nbsp;`nationalityCode` | `string` | ISO country code, eg `'CH'`
&nbsp;&nbsp;`sensitive` | `boolean` | sensitive flag. `true` if the person is sensitive and only people with the "Personen Admin" role can mutate that record afterwards.
&nbsp;&nbsp;`rating` | `string` | defines the creditor rating of this person.
&nbsp;&nbsp;`modeOfDispatch` | `string` | defines the mode of dispatch for this person. One of the following values will be accepted: `email`, `post`
&nbsp;&nbsp;`sendEMail` | `string` |  email address where the person wants to receive documents if `modeOfDispatch` is set to `email`

> &nbsp;&nbsp;`isCreditor` | `boolean` | declares whether this person has a creditor profile. `true` if they should have a creditor profile and `false` if they shouldn't.

Reviewer: ```suggestion
&nbsp;&nbsp;`isCreditor` | `boolean` | declares whether this person has a creditor profile.
```

> &nbsp;&nbsp;`creditorProfileIsBlocked` | `boolean` | declares whether this person's creditor profile is blocked. `true` if it should be blocked and `false` if it shouldn't be blocked.

Reviewer: ```suggestion
&nbsp;&nbsp;`creditorProfileIsBlocked` | `boolean` | declares whether this person's creditor profile is blocked.
```

&nbsp;&nbsp;`discount` | `decimal` | discount for this person

> &nbsp;&nbsp;`skonto` | `decimal` | skonto for this person. Must be between 0 and 100

Reviewer: ```suggestion
&nbsp;&nbsp;`skonto` | `decimal` | skonto for this person in percent.
```

&nbsp;&nbsp;`skontoInfo` | `string` | skonto information for this person.
&nbsp;&nbsp;`skontoDays` | `integer` | skonto information for this person. Must be empty, 0 or a positive integer

> &nbsp;&nbsp;`creditorIndustry` | `string` | creditor industry code for this person. Must be a valid creditor industry code (see code table entries "Kreditorbranchen" for valid codes).

Reviewer: Nice mention of the table. Certainly helps when using this.

    "maritalStatus":"married",
    "salutation":"SIR",
    "address":{
      "city":"Bern",
      "zipCode":"3007",
      "street":"Gartenstrasse 1/3",
      "countryCode":"CH"
    }
  }
}
```

##### accepted response message

```json
{"eventType":"Masterdata.Person.CreateAccepted",
  "data":{
    "personReference":"123456"
  }
}
```

##### rejected response message

```json
{"eventType":"Masterdata.Person.CreateRejected",
  "data":{
    "reasons":[
      {
        "attribute":"surname",
        "reason":"darf nicht leer sein"
      }
    ]
  }
}
```

### Masterdata.Person.Update

This message is sent from an external message publisher to a GARAIO REM instance and allows to update contact data of a person.
Set the recipient property in the headers, eg `"grem_wincasa"`. All attributes are optional unless noted otherwise in the remarks.

GARAIO REM replies with a standard [Accepted](./result_messages.md#accepted-message) / [Rejected](./result_messages.md#rejected-message) message containing the personReference and reject reasons, where appropriate

Field | Type | Content / Remarks
---|---|---
`eventType` | `string` | `Masterdata.Person.Update`
`data` | `hash` |

> &nbsp;&nbsp;`personReference` | `string` | reference of the person that should receive the communication updates; **required**

Reviewer: I am lazy and assuming the feedbacks from .Create apply for .Update too :)
